libsolutil/UTF8.cpp

```diff
@@ -76,8 +76,6 @@ bool isWellFormed(unsigned char byte1, unsigned char byte2)
 	return false;
 }
 
-}
-
 bool validateUTF8(unsigned char const* _input, size_t _length, size_t& _invalidPosition)
```

Comment:
Do we want to expose this in the "public api", or are we happy with the `std::string` only version?

Reply:
What we should do is expose a single ``std::string_view`` version of it :-).

Reply:
Sure, but not the point of this PR.

Reply:
Sure, didn't want to suggest that. Only takes a minute, I can create a PR in a second...